ArduCopter/Copter.h

```diff
@@ -588,6 +588,15 @@ class Copter : public AP_Vehicle {
     static const AP_Param::Info var_info[];
     static const struct LogStructure log_structure[];
 
+    // enum for ESC CALIBRATION
+    enum ESCCalibrationModes : uint8_t {
```

Comment:
Since you added ESCCalibrationModes everywhere, you can make this an `enum class [...]`.

Reply:
@OXINARF, thanks for having a look.  I did actually try making it an enum class but then it starts causing lots of compiler issue with casting between the parameter (which is an int8) and the enum.  I think it's more trouble than it's worth so I'm leaving it as a regular enum for now at least.